static/js/src/openstack/react/components/CostCalculations.jsx

```diff
@@ -0,0 +1,293 @@
+import React from "react";
+import { Row, Col } from "@canonical/react-components";
+
+import TCO_VARIABLES from "../utils/variables";
+
+const CostCalculations = (formState) => {
```

Comment:
```suggestion
const CostCalculations = ({ instances, vcpus, emepheralStorage, ram, persistentStorage, supportLevel}) => {
```
In here I think you could access formstate attribute directly (and I think eslint would require you to add proptypes for each of the parameters)